chromadb/config.py

```diff
@@ -124,7 +130,7 @@ def empty_str_to_none(cls, v: str) -> Optional[str]:
     chroma_server_ssl_enabled: Optional[bool] = False
 
     chroma_server_ssl_verify: Optional[Union[bool, str]] = None
-    chroma_server_api_default_path: Optional[str] = "/api/v1"
+    chroma_server_api_default_path: Optional[str] = APIVersion.V2
```

Comment:
if someone specifies an invalid config will this error?

Reply:
Updated to be of type `APIVersion`, now it should give a typing if they don't use the enum